recipe/meta.yaml

```diff
@@ -10,20 +10,21 @@ source:
   sha256: 9a7c7e922b87db3686210cf49d5d767033a41d4010b284e747682c92bddd8b39
 
 build:
-  noarch: python
```

Comment:
Please keep this, `importlib_metadata` is an empty package on newer Python versions.